cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -258,6 +259,7 @@ export default class SetPreferenceModal extends Component {
     event.preventDefault();
     this.setState({
       showResetMessage: true,
+      error: undefined,
```

Comment:
does `null` work? It's better to set to null than undefined. Undefined can imply that the property doesn't exist

Reply:
Fixed